guides/common/modules/con_configuring-external-authentication.adoc

```diff
@@ -4,6 +4,11 @@
 By using external authentication you can derive user and user group permissions from user group membership in an external identity provider.
 When you use external authentication, you do not have to create these users and maintain their group membership manually on {ProjectServer}.
 
+[NOTE]
+====
+In case the external source does not provide email, it will be requested during the first login through web UI.
```

Comment:
Please use proper attribute instead of "web UI".

Also, I don't think this line justifies a "note" box.

Reply:
Thank you for your feedback. Incorporated the suggested changes.